recipes/libmatio/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+./autogen.sh
+./configure --quiet --enable-shared --enable-mat73 --enable-extended-sparse --prefix="${PREFIX}" --with-zlib="${PREFIX}" --with-hdf5="${PREFIX}"
+make
+make install
```

Comment:
You can add an if clause to skip `make check` on OS X for now or check if you can remove whatever is conflicting with the checking from the OS X VM. I believe that @jakirkham is experimenting with removing some of homebrew pre-installed stuff that was messing with some tests in another lib.